t/lib/PAUSE/TestPAUSE.pm

```diff
@@ -355,13 +356,25 @@ sub test_reindex {
         for @{ $arg->{pick} };
     }
 
+    my sub filestate ($file) {
+      return ';;' unless -e $file;
+      my @stat = stat $file;
+      return join q{;}, @stat[0,1,7]; # dev, ino, size
```

Comment:
I care about "is different".  Normaly — in fact in a work commit recently! — I'd say "only check inode, because we make sure the writes are atomic".  Here, I'm just not super confident that I can say that!  So, I want to check content.  Obviously, as you say, shasum.  But the PAUSE tests use Path::Class, not Path::Tiny, so they don't have trivial access to that.  I have added #526 to track this.

Reply:
Thanks